### DISCUSSION TITLE
YouTube-like headers

### DISCUSSION THREAD

**Discussion by DabDatBass** [ScratchAddons/ScratchAddons#1028] [Ideas]
### Pros:

1. You can express yourself more
2. Have your profile page look cooler
3. You can use your signature banner in your profile
4. Cool backgrounds for your profile 👏 

### Cons:

1. Not everyone has Scratch Addons
2. Profile page banner is like this but for featured projects
3. Inappropriate images

_What do you think?_

**Comment by jeffalo** [comment]
![image](https://user-images.githubusercontent.com/40470736/102755131-4baca000-436e-11eb-81c8-770da96a042c.png)
![image](https://user-images.githubusercontent.com/40470736/102755169-523b1780-436e-11eb-903e-a6f6ccf19460.png)
basically already exists!

**Reply by daniel4-scratch** [reply]
Wow, neat

**Reply by cobaltt7** [reply]
> **Profile page banner is like this but for featured projects**

he means a different image

**Reply by DabDatBass** [reply]
Yea, so basically instead of a featured project and a gradient effect on your image you get a cool banner.

**Reply by daniel4-scratch** [reply]
This can’t be moderated by the scratch team if we have an external image uploading server

**Comment by cobaltt7** [comment]
maybe this goes in issues?

**Comment by scratchusernamemrtbts** [comment]
Oh i think you mean make the banner customizable instead of the featured project thumbnail

**Comment by cobaltt7** [comment]
Just use featured project banner and make your featured project look good with it on
________________________________
